src/styles/palette.js

```diff
@@ -64,6 +65,18 @@ export default function createPalette({
   error = red,
   type = 'light',
 } = {}) {
+  if (difference(keys(indigo), keys(primary)).length) {
+    throw new Error('Invalid primary color');
```

Comment:
Should we output the shape of the object expected? Maybe we could forward indigo too. Or we could let users looking at the source code.